src/index/gpu_raft/gpu_raft_ivf_flat_config.h

```diff
@@ -0,0 +1,75 @@
+/**
+ * SPDX-FileCopyrightText: Copyright (c) 2023,NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GPU_RAFT_IVF_FLAT_CONFIG_H
+#define GPU_RAFT_IVF_FLAT_CONFIG_H
+
+#include "common/raft/integration/raft_knowhere_config.hpp"
+#include "common/raft/proto/raft_index_kind.hpp"
+#include "index/ivf/ivf_config.h"
+#include "knowhere/config.h"
+
+namespace knowhere {
+
+struct GpuRaftIvfFlatConfig : public IvfFlatConfig {
+    CFG_FLOAT refine_ratio;
+    CFG_INT kmeans_n_iters;
+    CFG_FLOAT kmeans_trainset_fraction;
+    CFG_BOOL adaptive_centers;
+    KNOHWERE_DECLARE_CONFIG(GpuRaftIvfFlatConfig) {
+        KNOWHERE_CONFIG_DECLARE_FIELD(refine_ratio)
+            .set_default(1.0f)
+            .description("search refine_ratio * k results then refine")
+            .for_search();
+        KNOWHERE_CONFIG_DECLARE_FIELD(k)
+            .set_default(10)
+            .description("search for top k similar vector.")
+            .set_range(1, 256)  // Declared in base but limited to 256
```

Comment:
Please confirm that it is `(1, 256)` and that the upper bound is not higher (like 1024)

Reply:
raft ivf has this limits.